Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.39056&attempt_id=d2ff23ee-20a3-4cc1-8393-db8ede115a5f)

Attempted: jumanpp

<details><summary>Partial log (click to expand)</summary><p>

```
g++: error: unrecognized command line option '-m64'
make[3]: *** [Makefile:486: charlattice.o] Error 1
make[3]: Leaving directory '/build/jumanpp-1.02/src'
make[2]: *** [Makefile:519: all-recursive] Error 1
make[2]: Leaving directory '/build/jumanpp-1.02/src'
make[1]: *** [Makefile:347: all] Error 2
make[1]: Leaving directory '/build/jumanpp-1.02/src'
make: *** [Makefile:473: all-recursive] Error 1
builder for '/nix/store/712x4wx1ndx0vsbcz9anly1pwwzizg7k-jumanpp.drv' failed with exit code 2
[31;1merror:[0m build of '/nix/store/712x4wx1ndx0vsbcz9anly1pwwzizg7k-jumanpp.drv' failed
```
</p></details>

--- a/pkgs/tools/text/jumanpp/default.nix
+++ b/pkgs/tools/text/jumanpp/default.nix
@@ -17,6 +17,6 @@ stdenv.mkDerivation rec {
     homepage = http://nlp.ist.i.kyoto-u.ac.jp/index.php?JUMAN++;
     license = licenses.asl20;
     maintainers = with maintainers; [ mt-caret ];
-    platforms = platforms.all;
+    platforms = platforms.x86_64;
   };
 }

--- a/pkgs/tools/text/jumanpp/default.nix
+++ b/pkgs/tools/text/jumanpp/default.nix
@@ -19,6 +19,6 @@ stdenv.mkDerivation rec {
     homepage = http://nlp.ist.i.kyoto-u.ac.jp/index.php?JUMAN++;
     license = licenses.asl20;
     maintainers = with maintainers; [ mt-caret ];
-    platforms = platforms.x86_64;
+    platforms = platforms.all;
   };
 }

--- a/pkgs/tools/text/jumanpp/default.nix
+++ b/pkgs/tools/text/jumanpp/default.nix
@@ -1,12 +1,14 @@
-{ stdenv, fetchurl, boost, libunwind, gperftools }:
+{ stdenv, fetchurl, cmake, protobuf }:
 stdenv.mkDerivation rec {
   name = "jumanpp";
-  version = "1.02";
+  version = "2.0.0-rc2";
+
   src = fetchurl {
-    url = "http://lotus.kuee.kyoto-u.ac.jp/nl-resource/jumanpp/jumanpp-${version}.tar.xz";
-    sha256 = "14768b419bak8h2px8chw4cbfscb0jj012bpr769qv5nn6f53yh1";
+    url = "https://github.com/ku-nlp/${name}/releases/download/v${version}/${name}-${version}.tar.xz";
+    sha256 = "17fzmd0f5m9ayfhsr0mg7hjp3pg1mhbgknhgyd8v87x46g8bg6qp";
   };
-  buildInputs = [ boost libunwind gperftools ];
+  buildInputs = [ cmake protobuf ];
+
   meta = with stdenv.lib; {
     description = "A Japanese morphological analyser using a recurrent neural network language model (RNNLM)";
     longDescription = ''

